### DISCUSSION TITLE
Error : Your current working directory is in a git repository || Moose Installation

### DISCUSSION THREAD

**Discussion by stheeda07** [idaholab/moose#17645] [Q&A Getting Started] [answered]
I have installed moose and it passed all the tests. When I execute these commands,
$ cd ~/projects
$ ./moose/scripts/stork.sh cat

there is an error : your current working directory is in a git repository

I am unable to run stork.sh file to create an application. Thanks in advance.

**Comment by milljm** [comment, marked as answer]
What does the following report?

```bash
cd ~/projects
git worktree list
```

It sounds as if your projects directory (or perhaps the parent directory) is a git repo. I would like to understand more before we simply 'remove' this offending git repo. I know some use git as a means to back up everything...

**Reply by stheeda07** [reply]
Hey, git worktree list is returning : Users/stheeda  0000000 [master].

**Reply by stheeda07** [reply]
@milljm searched in google and did execute this command "**_rm -rf .git_**" in the projects directory. Now It works fine. But I don't know if this is 'legit'

**Reply by cticenhour** [reply]
@stheeda07 it does fix your immediate problem, if you didn't intend to make your home directory a git repository. If you did want it to be a git repository, well now the history associated with git is no longer available, unless there was some sort of backup elsewhere.

**Reply by stheeda07** [reply]
@cticenhour I never wanted my home directory to be a git repository. May be something might've gone wrong during git setup (not sure though). Anyways I have a backup of all the files on my parent directory
